api/requirements/prod.txt

```diff
@@ -21,4 +21,5 @@ nltk==3.4.5
 inflect
 werkzeug==0.16.1
 pysolr
-git+https://github.com/bcgov/namex-synonyms-api-py-client.git#egg=swagger_client
\ No newline at end of file
+git+https://github.com/bcgov/namex-synonyms-api-py-client.git#egg=swagger_client
+git+https://github.com/bcdevlucas/sbc-pay-py-client.git@dev#egg=openapi_client
```

Comment:
This has not been moved over yet to the bcgov repo?  It can wait til after rebase but just wondering.